assets/js/googlesitekit/modules/datastore/settings.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * `core/modules` data store settings
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+const { createRegistrySelector } = Data;
+
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+
+export const actions = {
+	/**
+	 * Submits all changes for a module.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {string} slug Slug for module store.
+	 * @return {Object} Module's submitChanges response object if it exists, otherwise object with `error` property if it doesn't.
+	 */
+	*submitChanges( slug ) {
+		invariant( slug, 'slug is required.' );
+		const registry = yield Data.commonActions.getRegistry();
+		if ( !! registry.dispatch( `modules/${ slug }` ) && !! registry.dispatch( `modules/${ slug }` ).submitChanges ) {
```

Comment:
This is a good use case for conditional chaining
```suggestion
		if ( !! registry.dispatch( `modules/${ slug }` )?.submitChanges ) {
```

Reply:
Addressed.